### DISCUSSION TITLE
Rotate an area

### DISCUSSION THREAD

**Discussion by JOKeRiino** [replit/kaboom#597] [Q&A] [answered]
So I have a laser beam going between two points. If the player hits the beam, he gets damaged. The laser beam can have an angle depending on the coordinates of the two points it connects. The area however is always horizontal. Is there a way to angle the area at the same angle the laser beam is at? tyia

![Bildschirmfoto 2022-06-09 um 14 12 10](https://user-images.githubusercontent.com/72083489/172843934-1893d0ad-44d4-4943-a04d-bc76ab08b3fb.png)

**Comment by lajbel** [comment, marked as answer]
Already added in `@next` (3000 alpha) :D 

npm i kaboom@next
